### PR TITLE
Add support to save npz files on s3

### PR DESCRIPTION
**Pull request by kayibal**

### CODE REVIEW

File: sparsity/test/test_sparse_frame.py [hunk 1]
```diff
@@ -19,6 +22,27 @@
     traildb = False
 
 
+@contextmanager
+def mock_s3_fs(bucket, data=None):
```
Comment (michcio1234) [inline sparsity/test/test_sparse_frame.py]: @kayibal Could you please add a docstring?

**Comment by kayibal** [comment]
@talmaj no I didn't but I don't see why this should be different from the tests?

**Comment by Talmaj** [comment]
S3 has sometimes problems with very large files, then they suggest using multi-part upload.

File: sparsity/test/test_sparse_frame.py (hunk 1 above)
Comment (kayibal) [inline sparsity/test/test_sparse_frame.py]: @michcio1234 added :)

**Pull request merged**